cypress/integration/login/login.spec.ts

```diff
@@ -48,8 +48,11 @@ describe('Test login page', () => {
         statusCode: 200,
         body: { }
       }).as('stats-request')
+
       // Login
-      cy.login(accountFixtures.default.username, accountFixtures.default.password)
+      const webuiUserName = Cypress.env('WEBUI_USERNAME') || accountFixtures.default.username
```

Comment:
where are these env set? they should be in the cypress.json yeah?

Reply:
I wrote them as an 'optional' because I have different local webui username password. I use cypress.env.json with those values. The || on line 54 still accommodate existing beahvior. That's why .gitignore has .cypress.env.*

Reply:
I did see what I thought was a mis-match between fixture values and usage of MPS .env values, but this part was totally out of scope for my story, I just needed some extra help to get the test to run in my environment so wanted to make it work for me, but also keep working the same way it was before with only making minimal changes.

Reply:
yeah i think thats fine if you wanna have your own cypress env, Just need to change it to match it up to override the mpsusername and mpspassword,